packages/server/src/plugin/subscriptionCallback/index.ts

```diff
@@ -218,7 +225,6 @@ class SubscriptionManager {
   > = new Map();
```

Comment:
I’m expecting a significant change in this `Map`'s data structure. In its current form, this `Map` captures one `heartbeat` per callback url. In order to have one `heartbeat` per subscription, we’d need to see `heartbeat` move into the `SubscriptionObject` type, along with all of the necessary code changes that would follow that change. Does that make sense? I don’t see how this can work in its current form without that change in the types contained here.